labs/lab3/chapter3.md

```diff
@@ -240,7 +240,7 @@ Note: See the difference in SELinux context after running w/ a volume & :Z.
 ls -lZd /var/lib/wp_uploads
 docker logs $(docker ps -ql)
 docker ps
-curl -L http://cdk.example.com:1080
+curl -L http://<YOUR AWS VM PUBLIC DNS NAME HERE>:1080
```

Comment:
yup, localhost works.  i don't mind either way.  i did test localhost.  this allows acces from desktop.